quot/helper.py

```diff
@@ -341,6 +341,8 @@ def threshold_image(I, t=200.0, return_filt=False, mode='max'):
     """
     I_bin = I > t
     pos = label_spots(I_bin, intensity_img=I, mode=mode)
+    if pos.size == 0:
+        pos = pos.reshape((0, 2))
```

Comment:
Unrelated bug fix. Output would formerly be shape (0,) if zero spots were identified; here we instead return (0, 2) to stay shape-consistent with the output with >0 spots.